docs/pages/feature-preview/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Feature Preview
+---
+
+The feature preview of the documentation is oriented towards new things which were added to the Expo ecosystem.
+
+Features available as a preview:
+
+- Expo Application Services. [Learn more](/eas)
+- Development Client. [Learn more](/clients/introduction)
```

Comment:
Previously, when a user selects `Feature Preview`, we link into `eas/index`.  I've changed that because this section isn't only about the EAS anymore. However, I didn't know what to put in this file. Do you have some ideas?

Reply:
Good thought!
I think we could:
- do some expectation setting around what Feature Preview means (generalizing the language in https://docs.google.com/document/d/1N4cxrsaHYCOPntycMDQ6HtEEJlKeQURTxw_xRDH1sHw/edit?usp=sharing)
- Break out build and submit (I expect EAS won't all leave preview at the same time)
- and add a one or two line description on what each feature is

As an aside, I'm wondering if config plugins belong here too?

Reply:
> Break out build and submit
> ...
> and add a one or two line description on what each feature is

yeah we could inline the same bullets from https://docs.expo.io/eas/ here and put a similar type of description for development client.

> do some expectation setting around what Feature Preview means

want to give that a shot?

Reply:
I made some adjustments 😉